lib/teleterm/daemon/daemon_test.go

```diff
@@ -453,7 +454,7 @@ func TestRetryWithRelogin(t *testing.T) {
 			}
 			require.Equal(t, tt.wantFnCalls, fnCallCount,
 				"Unexpected number of calls to fn")
-			require.Equal(t, tt.wantReloginCalls, service.callCounts["Relogin"],
+			require.EqualValues(t, tt.wantReloginCalls, service.reloginCount.Load(),
```

Comment:
Note: `require.Equal` does not allow comparing `int` to `uint32`.

Reply:
FYI you can cast the values too, but this is fine as well.